Description: We needed to check for more possibilities for
 where the files were installed.
Author: Steve Conklin <steve@conklinhouse.com>

--- d-rats-0.3.3.orig/d_rats/platform.py
+++ d-rats-0.3.3/d_rats/platform.py
@@ -177,6 +177,10 @@ class UnixPlatform(Platform):
             return "/usr/share/d-rats"
         elif "dist-packages" in find_me():
             return "/usr/share/d-rats"
+        elif "pymodules" in find_me():
+            return "/usr/share/d-rats"
+        elif "pyshared" in find_me():
+            return "/usr/share/d-rats"
         elif "/usr/share/d-rats" in find_me():
             return "/usr/share/d-rats"
         else:
